zc_utils.py

```diff
@@ -0,0 +1,19 @@
+import struct
+
+def write_compact_size(n):
+    if n < 253:
+        return struct.pack('B', n)
+    elif n < 0xFFFF:
```

Comment:
This is not correct; it is `<= 0xFFFF` (see https://bitcoin.org/en/developer-reference#compactsize-unsigned-integers ).